torch/_tensor_docs.py

```diff
@@ -3202,10 +3238,39 @@ def callable(a, b) -> number
                r"""
 sparse_dim() -> int
 
-If :attr:`self` is a sparse COO tensor (i.e., with ``torch.sparse_coo`` layout),
+If :attr:`self` is a :ref:`sparse tensor <sparse-docs>`,
 this returns the number of sparse dimensions. Otherwise, this throws an error.
 
-See also :meth:`Tensor.dense_dim`.
+See also :meth:`Tensor.dense_dim` and :ref:`hybrid tensors <sparse-hybrid-docs>`.
+""")
+
+add_docstr_all('sparse_resize_',
+               r"""
+sparse_resize_(size, sparse_dim, dense_dim) -> Tensor
+
+Resizes :attr:`self` :ref:`sparse tensor <sparse-docs>` to the
+desired (non-shrinking) size and the number of sparse and dense
+dimensions.
+
+Args:
+    size (torch.Size): the desired size.
+    sparse_dim (int): the number of desired sparse dimensions
```

Comment:
"the number of sparse dimensions" is fine

Reply:
fixed.